design-proposals/export-import/volume-export.md

```diff
@@ -0,0 +1,161 @@
+# Overview
+Proposal for exporting KubeVirt Virtual Machine volumes from a Kubernetes cluster
+
+## Motivation
+There are currently multiple ways to import/initialize Virtual Machine volumes.  But there is no easy and efficient method to get volume data out of the cluster.
+
+## Goals
+* Define VirtualMachineExport API
+* Introduce virtctl export command
+
+## Non Goals
+* Implementation details of internal components (servers/proxies/controllers/etc)
+
+## User Stories
+* As a KubeVirt user, I would like to export a Virtual Machine volume for local debugging/analysis.
```

Comment:
> export a Virtual Machine volume

Is this proposal about exporting a single volume? I think that the more interesting case is about exporting all VM data: its spec, multiple volumes and possibly RAM.

Reply:
The idea is that `VirtualMachineExport` will make it possible to export all persistent data (including mem dumps) related to a VirtualMachines.  Manifests are already available via k8s apiserver.  Could be added here but I think it would be better to rely on standard k8s API/RBAC for that.

Reply:
I'm not sure I understand your answer. The propose text speak about a (single) Virtual Machine volume. While I think it should speak about a virtual machine definition and all associated data.